README.md

```diff
@@ -21,9 +21,14 @@ You can use the resulting tokens with any service that supports JWT tokens.
 To run the service you need to create the configuration that contains all the needed keys and identifiers.
 To do this, generate a fresh DID for the service deployment and then create a config file from it.
 
+First assign your KILT account seed to a variable named SEED then run opendid-setup container.
+
+`SEED="dont try this seed its completely made up for this nice example"`
+
+If your account is on KILT Spiritnet, ensure that you include `ENDPOINT=spiritnet` environment variable; otherwise, the command will automatically default to Peregrine testnet chain.
+
 ```bash
-SEED="dont try this seed its completely made up for this nice example"
-podman run --rm -it -v $(pwd):/data docker.io/kiltprotocol/opendid-setup:latest "${SEED}"
+podman run --rm -it -e "ENDPOINT=spiritnet" -v $(pwd):/data docker.io/kiltprotocol/opendid-setup:latest "${SEED}"
```

Comment:
when do we get rid of podman here? I think there is not really a reason besides Tino's personal preference to use it instead of docker. And docker is the more widely known tool by far

Reply:
:wave: podmam